107/src/test/resources/ehcache-107-integration.xml

```diff
@@ -9,8 +9,7 @@
     </jcache:defaults>
   </ehcache:service>
 
-  <ehcache:cache alias="productCache"
-                 storeByValueOnHeap="true"> <!-- OPTIONAL: storing by value on heap, will result in keys and values being copied in and out of the Cache -->
```

Comment:
Your changes make this attribute obsolete. But could you update the test so that it still "creates" a cache consider byValue in 107 world?